Makefile

```diff
@@ -10,7 +15,7 @@ safety: ## Runs `safety check` to check python dependencies for vulnerabilities
 
 .PHONY: update-pip-requirements
 update-pip-requirements: ## Updates all Python requirements files via pip-compile.
-	pip-compile --generate-hashes --output-file requirements/test-requirements.txt requirements/test-requirements.in
+	pip-compile --allow-unsafe --generate-hashes --output-file requirements/test-requirements.txt requirements/test-requirements.in
```

Comment:
Nit: do we want `--upgrade` here? These are test reqs, so running the target should always go-get-the-latest, right?